configuration/configuration.go

```diff
@@ -133,7 +140,15 @@ func LoadConfiguration(baseDirectory string) (*Configuration, error) {
 		Depth:     pruneDepth,
 		MinHeight: minPruneHeight,
 	}
-
+	maxSyncValue := os.Getenv(MaxSyncConcurrency)
+	switch maxSyncValue {
+	case "":
+		config.MaxSyncConcurrency = syncer.DefaultMaxConcurrency
+	case "0":
+		return nil, errors.New("syncer concurrency must be greater than zero")
+	default:
+		config.MaxSyncConcurrency, _ = strconv.ParseInt(maxSyncValue, 10, 64)
```

Comment:
you should explicitly handle this parsing error

Reply:
Handled in new commit.

Note: Allowed for MAXSYNC to not be set and default to 256